src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -379,11 +392,15 @@ const WorkspaceDashboard = _.flow(
         ]),
         Utils.canWrite(accessLevel) && h(InfoRow, {
           title: 'Estimated Storage Cost',
-          subtitle: !!storageCost ? `Updated on ${new Date(storageCost.lastUpdated).toLocaleDateString()}` : 'Loading last updated...'
+          ...(storageCost?.isSuccess ?
+            { subtitle: `Updated on ${new Date(storageCost.lastUpdated).toLocaleDateString()}` } :
+            !storageCost && { subtitle: 'Loading last updated...' })
```

Comment:
```suggestion
          subtitle: Utils.cond(
            [!storageCost, () => 'Loading last updated...'],
            [storageCost.isSuccess, () => `Updated on ${new Date(storageCost.lastUpdated).toLocaleDateString()}`]
          )
```

And similar for the "Bucket Size" row.